account_claims.go

```diff
@@ -184,3 +186,25 @@ func (a *AccountClaims) DidSign(op Claims) bool {
 	}
 	return false
 }
+
+// RevokeUser enters a revocation by publickey and timestamp into this export
+// If there is already a revocation for this public key that is newer, it is kept.
+func (a *AccountClaims) RevokeUser(pubKey string, timestamp time.Time) {
+	if a.UserRevocations == nil {
+		a.UserRevocations = RevocationList{}
+	}
+
+	a.UserRevocations.Revoke(pubKey, timestamp)
+}
+
+// ClearUserRevocation removes any revocation for the public key
+func (a *AccountClaims) ClearUserRevocation(pubKey string) {
+	a.UserRevocations.ClearRevocation(pubKey)
+}
+
+// IsUserRevoked checks if the public key is in the revoked list with a timestamp later than
+// the one passed in. Generally this method is called with time.Now() but other time's can
+// be used for testing.
+func (a *AccountClaims) IsUserRevoked(pubKey string, timestamp time.Time) bool {
+	return a.UserRevocations.IsRevoked(pubKey, timestamp)
```

Comment:
Wonder if we should simplify the case where time is time.Now and have an additional method that can take time.

Reply:
thought about this, but in cases where you are checking > 1 it seems useful to call time.Now() once.

Reply:
but - i am going to add an IsUserRevokedNow and an IsRevokedNow() to the other place, that way we can use what is more convenient

Reply:
It feels like semantically that IsRevoked() is binary. Yes we could ask about time in the past but not sure that makes sense for most scenarios. 

Also time.Now() is not too penal, and I doubt we will be asking it alot from a context where we could share the time.Now().